src/IO/ITKImageInputFileFormat.cxx

```diff
@@ -400,26 +445,33 @@ struct removePtr<itk::SmartPointer<Type> >
   typedef Type type;
 };
 
+template<typename STIRImageType>
 CartesianCoordinate3D<float>
-ITK_coordinates_to_STIR(
-        const itk::ImageBase<3>::PointType &itk_coord,
-        CartesianCoordinate3D<float> &voxel_size,
-        const BasicCoordinate<3,int> &min_indices,
-        bool is_displacement_field)
+ITK_coordinates_to_STIR(const itk::ImageBase<3>::PointType &itk_coord,
+                        const STIRImageType stir_image,
```

Comment:
reference to `STIRImageType`?

Reply:
`const STIRImageType& stir_image`? Sure, but  curious why?

Reply:
yes. (potentially) big objects should always be passed by (preferably `const`) reference. Note for stuff like `int`, `float` etc. Really should add this to our developers-guide...